collectors/statsd.plugin/k6.md

```diff
@@ -0,0 +1,72 @@
+<!--
+title: "K6 Load Testing monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/stats.d.plugin/k6.md
+sidebar_label: "K6 Load Testing"
+-->
+
+# K6 Load Testing monitoring with Netdata
+
+Monitors the performance of your Load Testing experiments using [K6](https://k6.io/).
```

Comment:
I feel like "impact" is maybe the more appropriate word here. I also changed "using," because I worry that people might think you use K6 to do the monitoring. Silly, I know, but just to be safe.

```suggestion
Monitors the impact of load testing experiments performed with [K6](https://k6.io/).
```